### PR TITLE
Wip replace diem logos jh

### PR DESCRIPTION
**Pull request by moonclavedev**
## Motivation

Initial sweep of Removing Diem logos from Charts SVG files

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

Check Docs for Chart images that may still display Diem logo.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site, and link to your PR here.)

## If targeting a release branch, please fill the below out as well

 * Justification and breaking nature (who does it affect? validators, full nodes, tooling, operators, AOS, etc.)
 * Comprehensive test results that demonstrate the fix working and not breaking existing workflows.
 * Why we must have it for the release.
 * What workarounds and alternative we have if we do not push the PR.

### CODE REVIEW

**Comment by JoshLind** [comment]
> Initial sweep of Removing Diem logos from Charts SVG files

Thanks @moonclavedev! Is the only purpose of this PR to remove the logos from the files? What about the use of the word "Diem" in the images? There still seems to be a bunch of those in the drawings themselves, but I'm not sure if that's something we're wanting to handle here?

**Comment by moonclavedev** [comment]
> > Initial sweep of Removing Diem logos from Charts SVG files
> 
> Thanks @moonclavedev! Is the only purpose of this PR to remove the logos from the files? What about the use of the word "Diem" in the images? There still seems to be a bunch of those in the drawings themselves, but I'm not sure if that's something we're wanting to handle here?

Hey @JoshLind ! Yes, the initial request was to remove the logos - however you are right and we should also remove "Diem" in the copy as well. This is a bit more work as all text within the SVG's has been outlined (flattened into a Path) and will need to be recreated. 

Assuming we might not have original source files for the artwork with editable text, it's probably best to take this time to recreate all text layers within the SVG's. We can then change all references of "Diem" to "Aptos" and also allow for us to utilize a new font to match the new branding.

Thinking further out - we could explore options for inlining these SVG's (as opposed to ``) which would allow usage of `<text class=""></text>` within the markup. Any text within the SVG's would then be machine-readable and part of the codebase providing the following benefits: 

- Indexing the content for search engines 
- Accessibility / screen-reader concerns /
- Making quick edits on the fly without the need for image editing software
- Translations
- Targeting elements by class name with JS / CSS 

For now, I will begin recreating these images with new text layers as a **.ai** source which we can export to **.svg** (the way it is now), retaining the ability to make further changes and potentially consider the inline option in the future if it makes sense.

File: terraform/helm/validator/files/dashboards/components.json [hunk 1]
```diff
@@ -2021,7 +2021,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "aptos_core_mempool_index_size{role=~\"$role\",owner=~\"$owner\",index=\"system_ttl\"}",
+          "expr": "core_mempool_index_size{role=~\"$role\",owner=~\"$owner\",index=\"system_ttl\"}",
```
Comment (JoshLind) [inline terraform/helm/validator/files/dashboards/components.json:2024]: Note: I would have changed these counters in a separate PR -- makes the history easier to track 😄 But, no biggie!

**Comment by JoshLind** [comment]
Awesome, thanks @moonclavedev! 😄 

>  Yes, the initial request was to remove the logos - however you are right and we should also remove "Diem" in the copy as well. This is a bit more work as all text within the SVG's has been outlined (flattened into a Path) and will need to be recreated.

Cool, makes sense! Approved this so we can get the fixes in. No reason to block these changes while waiting on the other more complex ones 🚀 

> Assuming we might not have original source files for the artwork with editable text, it's probably best to take this time to recreate all text layers within the SVG's. 

Yeah -- that's probably the right strategy longer term! Otherwise, it'll be hacky trying to change these.

> I will begin recreating these images with new text layers as a .ai source which we can export to .svg (the way it is now), retaining the ability to make further changes and potentially consider the inline option in the future if it makes sense.

Awesome, thanks! This sounds great!

**Review by JoshLind — APPROVED** [review]

**Comment by CapCap** [comment]
please rebase your diff against main- it should remove the non-image changes to tf :-)

**Comment by davidiw** [comment]
Unfortunately we don't have access to the original sources :(

**Comment by davidiw** [comment]
https://github.com/aptos-labs/aptos-core/pull/277 let's maybe move this to an issue?

**Pull request closed without merging**